src/components/button/button.js

```diff
@@ -29,8 +31,9 @@ export default class Button extends PureComponent {
     disabled: PropTypes.bool,
     remove: PropTypes.bool,
     isStatic: PropTypes.bool,
+    isRounded: PropTypes.bool,
```

Comment:
Can you change the prop name to `rounded` the isStatic prop is an exception because static is a reserved word

Reply:
Sure my bad :)